static/src/js/components/SearchSidebar/__tests__/SearchSidebarHeader.test.js

```diff
@@ -117,7 +117,7 @@ describe('SearchSidebarHeader component', () => {
       })
 
       await waitFor(() => {
-        expect(screen.getByTestId('portal-logo-link').href).toEqual('https://idn.ceos.org/')
+        expect(screen.getByTestId('portal-logo-link').href).toEqual('https://ceos.org/ourwork/workinggroups/wgiss/access/international-directory-network/')
```

Comment:
This doesn't need to be done in this PR, but this test should mock a portal config, not use a real config, to avoid any kind of issue like this in the future. Portal config changes don't need to be changing this kind of file @eudoroolivares2016

Reply:
Okay I can do that or someone else can before this merges; @mmorahan that is pretty outside the scope of what you are trying to do with this config change and something one of us should do. I'll see how we wanna handle this ticket wise

Reply:
Wrote separate ticket to do this action `EDSC-4077`